parent/pom.xml

```diff
@@ -928,6 +936,14 @@
 				<enabled>true</enabled>
 			</snapshots>
 		</pluginRepository>
+                <pluginRepository>
+                        <id>redhat-maven-repository</id>
```

Comment:
this should not be necessary. we don't want to accidentally pick up maven plugins from this repo if can be avoided.